client/state/jetpack-settings/modules/test/reducer.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+import {
+	JETPACK_MODULE_ACTIVATE,
+	JETPACK_MODULE_ACTIVATE_FAILURE,
+	JETPACK_MODULE_ACTIVATE_SUCCESS
+} from 'state/action-types';
+import {
+	items as itemsReducer,
+	requests as requestsReducer
+} from '../reducer';
+
+import {
+	modules as MODULES_FIXTURE,
+	requests as REQUESTS_FIXTURE
+} from './fixture';
+
+describe( 'reducer', () => {
+	describe( 'items', () => {
+		it( 'state should default to empty object', () => {
+			const state = itemsReducer( undefined, {} );
+			expect( state ).to.eql( {} );
+		} );
+
+		describe( '#modulsActivation', () => {
```

Comment:
In the future there might be some actions that can't be classified as just one type of actions (modules activation). In that case we'd have to refactor these `describe` calls I'd remove that `describe`. But at this time it makes sense, so that's your call, really. 

Ah, and there's a typo - `modulsActivation` - `s` should be `e`.